integration/helpers/images.go

```diff
@@ -38,3 +40,19 @@ func UniqueName() string {
 	name := base32.StdEncoding.WithPadding(base32.NoPadding).EncodeToString(util.GenerateBytesUUID())
 	return strings.ToLower(name)
 }
+
+func ExistsInPath(cmd string) bool {
+	dirsInPath := strings.Split(os.Getenv("PATH"), fmt.Sprintf("%c", os.PathListSeparator))
+	for _, dir := range dirsInPath {
+		info, err := ioutil.ReadDir(dir)
+		if err != nil {
+			continue
+		}
+		for _, f := range info {
+			if f.Name() == cmd {
+				return true
+			}
+		}
+	}
+	return false
+}
```

Comment:
Can't we just use: https://golang.org/pkg/os/exec/#LookPath

Reply:
I was looking for something like this but couldn't find it, thanks.